core/store/presenters/presenters.go

```diff
@@ -462,6 +462,25 @@ func (sa ServiceAgreement) FriendlyPayment() string {
 	return fmt.Sprintf("%v LINK", sa.Encumbrance.Payment.String())
 }
 
+// FriendlyAggregator returns the ServiceAgreement's aggregator address,
+// in a human readable format.
+func (sa ServiceAgreement) FriendlyAggregator() string {
+	return fmt.Sprintf("%s", sa.Encumbrance.Aggregator)
```

Comment:
What about using .String()?  It is used by the 'fmt' package in this instance.

https://golang.org/pkg/fmt/#Stringer

Reply:
That's a better approach, yeah. Fixed in 061cbd39e1833963a427cea76405ed34dc9cf17d.